pipelines/pipelines/document_stores/elasticsearch.py

```diff
@@ -1297,6 +1299,7 @@ def query(
                     "The query provided seems to be not a string, but an object "
                     f"of type {type(query)}. This can cause Elasticsearch to fail."
                 )
+            operator = "AND" if all_terms_must_match else "OR"
```

Comment:
这里没有太明白这里的逻辑，all_terms_must_match 表示什么？ 建议这里加一下相关代码注释

Reply:
已添加：

:param all_terms_must_match: Whether all terms of the query must match the document.
                                     If true all query terms must be present in a document in order to be retrieved (i.e the AND operator is being used implicitly between query terms: "cozy fish restaurant" -> "cozy AND fish AND restaurant").
                                     Otherwise at least one query term must be present in a document in order to be retrieved (i.e the OR operator is being used implicitly between query terms: "cozy fish restaurant" -> "cozy OR fish OR restaurant").
                                     Defaults to false.